climada/util/checker.py

```diff
@@ -228,3 +228,32 @@ def prune_csr_matrix(matrix: sparse.csr_matrix):
     matrix.check_format()
     matrix.eliminate_zeros()
     matrix.sum_duplicates()
+
+
+def convert_frequency_unit_to_time_unit(frequency_unit):
+    """Converts common frequency units to corresponding time units. Unknown frequency
+    units are converted to "years".
```

Comment:
wouldn't the `climada.util.dates_times` package be a better match for this method?